lib/nx_crc.c

```diff
@@ -36,26 +36,17 @@
 #include <stdint.h>
 #include <stddef.h>  /* ptrdiff_t */
 #include <byteswap.h>
-#include "crc32_ppc.h"
+#include "nx_zlib.h"
 
 /* make crc32.c work without external zlib headers */
 #undef MAKECRCH
 #undef DYNAMIC_CRC_TABLE
-#define FAR
 #define local static
-#define OF(args)  ()
-typedef unsigned long uLong;
-typedef unsigned int uInt;
-typedef uint32_t z_crc_t;
 typedef size_t   z_size_t;
```

Comment:
I rebased on top of current develop and also re-added this line. Without the project doesn't compile on RHEL7 for the same reason as in 74b61ae